README.md

```diff
@@ -6,6 +6,8 @@
 
 This repository includes files with plain SQL that can be used to recreate a database:
 
+- Clone this repository by running the command below;
+  - git clone https://github.com/Loltolo-Lesapiti/clinic_database.git
```

Comment:
- [ ] Great job so far. However, all "join table" for the many-to-many relastionships must be created, even if they are not showed on the diagram. You should create a table between medical_histories and treatments  for the many-to-many relationship. Address this 👍🏻 
- eg:
```sql
  CREATE TABLE medical_histories_has_treatments (
    ...
    );
```